webservice/template/assignment/upload.tmpl

```diff
@@ -87,7 +88,7 @@
 
             // Output the result in an element with id="demo"
             document.getElementById("countDown").innerHTML = "<strong>Time left: </strong>" + days + "d " + hours + "h "
-                + minutes + "m " + seconds + "s ";
+                + minutes + "m " + seconds + "s " + "<i>(Norwegian time!)</i>";  <!-- TODO time-norwegian -->
```

Comment:
Remove "(Norwegian time!)"

Reply:
Christopher asked for this specific, so nah